web/struct_tree_layer_builder.js

```diff
@@ -156,6 +160,43 @@ class StructTreeLayerBuilder {
     }
   }
 
+  #addImageInTextLayer(node, element) {
+    const { alt, bbox, children } = node;
+    const child = children?.[0];
+    if (!this.#viewBox || !alt || !bbox || child?.type !== "content") {
+      return false;
+    }
+
+    const { id } = child;
+    if (!id) {
+      return false;
+    }
+    const span = document.getElementById(id);
+    if (!span) {
+      return false;
+    }
+
+    element.setAttribute("aria-owns", id);
+
+    const img = document.createElement("span");
+    span.append(img);
+    img.setAttribute("role", "img");
+    const label = removeNullCharacters(alt);
+    img.setAttribute("aria-label", label);
```

Comment:
Nit: Maybe just inline this?
```suggestion
    img.setAttribute("aria-label", removeNullCharacters(alt));
```